packages/admin-ui/src/Command/domain/CommandOptionDto.ts

```diff
@@ -0,0 +1,7 @@
+export interface CommandOptionDto {
+    label: string;
+    value: string;
+    disabled?: boolean;
+    separator?: boolean;
+    item?: any;
```

Comment:
Not sure if we want to go with `any` here. Any way to improve this? Like make this interface generic?

But ofc, not P1. If it's not worth it at this point, let's just backlog it.

Reply:
It is worth it, and I agree it is not a P1. 

I logged it...thank you for pointing this out 👍